plutarch-ledger-api/src/Plutarch/LedgerApi/V1/Time.hs

```diff
@@ -42,68 +47,35 @@ newtype PPosixTime (s :: S) = PPosixTime (Term s (PDataNewtype PInteger))
 
 -- | @since 2.0.0
 instance PIntegral PPosixTime where
+  {-# INLINEABLE pdiv #-}
   pdiv = phoistAcyclic $ plam $ \t1 t2 ->
-    pmatch t1 $ \(PPosixTime t1') ->
-      pmatch t2 $ \(PPosixTime t2') ->
-        pmatch t1' $ \(PDataNewtype t1'') ->
-          pmatch t2' $ \(PDataNewtype t2'') ->
-            pcon . PPosixTime . pcon . PDataNewtype . pdata $ pdiv # pfromData t1'' # pfromData t2''
+    pposixTime (pdiv # unPPosixTime t1 # unPPosixTime t2)
+  {-# INLINEABLE pmod #-}
```

Comment:
Why are you inlining this stuff

Reply:
I don't think this really matters. It's in-lining Haskell functions anyways. Might as well have better performing haskell functions